settings/prod_settings.py

```diff
@@ -41,6 +41,6 @@
 print(f"[BACKEND] Using {DB_TYPE} database: {os.environ.get('DATABASE_NAME')}")
 
 
-ALLOWED_HOSTS: list[str | None] = [os.environ.get("URL")] + URL_LIST
```

Comment:
Could this be left with | None as described in https://github.com/TreyWW/MyFinances/pull/324#discussion_r1563955230

Reply:
Yes, I'll put there type ignore then since it have problem concatenating `list[str | None]` with `list[str]`